### DISCUSSION TITLE
Benchmarking workflow

### DISCUSSION THREAD

**Discussion by anwarMZ** [cidgoh/bacpaq#74] [Announcements]
- [ ] Find a bacterial WGS & Amplicon (16S) nanopore dataset. 
Selection criteria: 
1. Preferably PromethION
- [ ] Run the Nanopore QC and Taxonomy QC workflow with the following modules
1. NANOCOMP
2. PYCOQC
3. KRAKEN2 +KRONA
4. De-hosting
- [ ] Time as limit and then find the time taken by each sample.

**Comment by jimmyliu1326** [comment]
Potentially convenient to have config files encoding parameter presets for different applications like isolate WGS, amplicon sequencing, etc. 

Analogous to the presets available in minimap2.

For example in long read isolate WGS, the standard practice is filter reads shorter than 1 kbps and dehosting is less common. Conversely in long read amplicon sequencing, we likely want to have dehosting, a less stringent read length cutoff and skip genome assembly.
